.github/workflows/release.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Release role
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  import_role:
+    name: Import role to Ansible Galaxy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.x'
+      - name: Install Ansible.
+        run: pip3 install ansible-core==2.17.*
+      - name: Start import on Ansible Galaxy
+        run: >
+          ansible-galaxy role import
+          --token ${{ secrets.ANSIBLE_GALAXY_API_TOKEN }}
+          $(echo ${{ github.repository }} | cut -d/ -f1)
+          $(echo ${{ github.repository }} | cut -d/ -f2)
```

Comment:
@sdclarke This looks good to me

Let's add a CI job to check the ansible correctness (so it runs when a PR is sent)

Reply:
Added a job to run ansible-lint on pull requests.